src/NServiceBus.RabbitMQ/Connection/ConnectionFactory.cs

```diff
@@ -46,11 +47,18 @@ public ConnectionFactory(ConnectionConfiguration connectionConfiguration)
             }
         }
 
+        public IConnection CreatePublishConnection() => CreateConnection("Publish");
```

Comment:
This is a change in behaviour since the `ConnectionManager` used to cache the publish connection. Is that a problem?